lib/handlers/runtimeHandler.js

```diff
@@ -53,6 +53,9 @@ const runtimeEvaluate = async (exp, executionContextId, opt = {}) => {
   try {
     runtimeResult = await runtime.evaluate(options);
   } catch (e) {
+    if (e.message.match(/Chrome process with pid \d+ exited with/)) {
```

Comment:
I am not sure if this will work if we change the browser langauge and internationalisation as we are matching messages in english. 

Can you check this after changing the browser or OS langauge?

Reply:
I tried after changing the langauge to french and every thing works fine.